go.mod

```diff
@@ -1,6 +1,10 @@
 module github.com/open-policy-agent/gatekeeper/v3
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.2
```

Comment:
will this downgrade Go version? do we need `GOTOOLCHAIN` set to `local` for release?

Reply:
removed `toolchain go1.22.2`